changelogs/unreleased/gh-5815-memprof-enrich-symtab.md

```diff
@@ -1,7 +1,7 @@
 ## feature/luajit
 
-* Now memory profiler emits events of the new type when a function or a trace
-  is created. As a result memory profiler parser can enrich its symbol table
-  with the new functions and traces (gh-5815). Furthermore, there are symbol
-  generations introduced within internal parser structure to handle possible
-  collisions of function addresses and trace numbers.
+* Now the memory profiler emits events of the new type when a function or a trace
+  is created. As a result the memory profiler parser can enrich its symbol table
+  with the new functions and traces (gh-5815). 
+  Furthermore, there are symbol generations introduced within the internal parser structure 
+  to handle possible collisions of function addresses and trace numbers.
```

Comment:
I guess your aim is to show those texts as separate paragraphs. Markdown requires an empty line between paragraphs. Look at 'View file' in the '...' menu in the pull request.